openverse_catalog/dags/providers/factory_utils.py

```diff
@@ -117,6 +117,9 @@ def pull_media_wrapper(
         ingester = ingestion_callable(*args)
         stores = ingester.media_stores
         run_func = ingester.ingest_records
+        # args have already been passed into the ingester, we don't need them passed
+        # in again to the ingest_records function, so we clear the list
+        args = []
```

Comment:
@stacimc do you think this is sufficient to cover that case?

Reply:
I'm fine with that -- it'll be easy to clean up anyway once the refactors are done :)